src/main/scala/li/cil/oc/client/renderer/tileentity/RobotRenderer.scala

```diff
@@ -214,6 +214,15 @@ object RobotRenderer extends TileEntitySpecialRenderer {
           RenderState.disableLighting()
         }
 
+        // Additive blending for the light.
+        GL11.glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE)
+        // Light color.
+        var lightColor = if (robot == null) 15872048 else if (robot.info == null) 15872048 else robot.info.lightColor
```

Comment:
Please use hex values for the colors, makes it much more readable (`0xF23030` in this case, which immediately makes it apparent that it's a red-ish tone).

Also, as Kubuxu said, simplify it to
```scala
if (robot != null && robot.info != null) robot.info.lightColor else 0xF23030
```